src/org/zaproxy/zap/extension/alertFilters/ContextAlertFilterPanel.java

```diff
@@ -103,7 +103,8 @@ public AlertFiltersMultipleOptionsPanel(ExtensionAlertFilters extension, AlertFi
 			super(model);
 			this.extension = extension;
 
-			getTable().getColumnExt(0).setMaxWidth(70);
+			getTable().getColumnExt(0).sizeWidthToFit();
```

Comment:
I could drop 106 (size to fit) and just let the max be however it's rendered by default (leaving line 107). Is that what you meant? (The call you quoted doesn't lead to a width constant or getWidth function...that I could find anyway.)